tests/test_fixture.py

```diff
@@ -59,6 +79,45 @@ def test_record_tuples(caplog):
     ]
 
 
+@pytest.mark.parametrize('name, level, message, expected', filter_params)
+def test_filter_records(caplog, name, level, message, expected):
+    do_test_filter_record_logging()
+
+    filtered = caplog.filter_records(name, level, message)
+    filtered_named_args = caplog.filter_records(name=name, level=level, message=message)
+
+    assert filtered == filtered_named_args
+    assert [f.getMessage().split(' ')[0] for f in filtered] == expected
+
+
+@pytest.mark.parametrize('name, level, message, expected', filter_params)
+def test_filter_record_tuples(caplog, name, level, message, expected):
+    do_test_filter_record_logging()
+
+    filtered = caplog.filter_record_tuples(name, level, message)
+    filtered_named_args = caplog.filter_record_tuples(name=name, level=level, message=message)
+
+    assert filtered == filtered_named_args
+    assert [f[2].split(' ')[0] for f in filtered] == expected
+
+
+@pytest.mark.parametrize('level', [
+    'CRITICAL',
+    'ERROR',
+    'WARNING',
+    'INFO',
+    'DEBUG',
+    'NOTSET',
+])
+def test_levels(caplog, level):
+    assert getattr(caplog, level) == getattr(logging, level)
+
+
+def test_levels_error(caplog):
+    with pytest.raises(AttributeError):
+        getattr(caplog, 'UNKNOWNLEVEL')
```

Comment:
Please, also add a check for lower case attributes.
Plus: `'LEVEL_WITH_UNDERSCORES'`, `LEVEL42`
Thanks